program/src/process_management.rs

```diff
@@ -38,12 +38,12 @@ pub fn process_change_reward_distribution(
     lido.fee_recipients.treasury_account = *accounts.treasury_account.key;
     lido.fee_recipients.developer_account = *accounts.developer_account.key;
 
-    lido.save(accounts.lido)
```

Comment:
The main point is that you need to have data reference `&mut [u8]` on stack to be able to use BigVec. So the function can't return `ValidatorList<'a>` because a temporary data reference `&mut [u8]` will be destroyed. I could think about a better interface but now it looks convenient to use in instructions.

Reply:
`split_at_mut` will create temp variables on stack that will be destroyed after `Lido::deserialize_lido` returns. So you can't call `lido_account.data.borrow_mut` in that function. You should split the data before calling it. And it looks ugly, I tried.
Two make everything beautiful I suppose we have to use BigVec with `data: Rc<RefCell<&'a mut u64>>`. But it is overdo.